fyde-app/linux.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+title: Linux
+parent: Fyde App
+nav_order: 1
+---
+# Fyde App - Linux
+
+Instructions to install the Fyde App in different distributions.
+
+There are two versions available:
+
+- fyde - complete GUI version (includes fyde-tunnel)
+- fyde-tunnel - CLI version
```

Comment:
I would write something like this:

Fyde App for Linux is distributed in two separate packages:

- fyde - GUI for interactive user sessions (depends on fyde-tunnel)
- fyde-tunnel - standalone network services and command line utilities designed to run in headless setups

Reply:
That's better indeed. Going to change.